src/apprt/gtk/ui/prompt-title-dialog.blp

```diff
@@ -0,0 +1,10 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.AlertDialog prompt_title_dialog {
+  heading: "Change Terminal Title";
+  body: "Leave blank to restore the default title.";
```

Comment:
Prefer [translatable strings](https://jwestman.pages.gitlab.gnome.org/blueprint-compiler/reference/values.html#translated-strings) for user-facing text